editor/node_3d_editor_gizmos.h

```diff
@@ -31,7 +31,7 @@
 #ifndef SPATIAL_EDITOR_GIZMOS_H
 #define SPATIAL_EDITOR_GIZMOS_H
```

Comment:
Should also change the include guards along with the class and file name.

Reply:
Too difficult to do consistently with all those renames, I'll write a script to do it (#37290).